experiment/builder_runner.py

```diff
@@ -538,7 +548,8 @@ def build_and_run_local(
         run_result.crashes, run_result.crash_info, \
           run_result.semantic_check = \
             self._parse_libfuzzer_logs(f, project_name, flag)
-      run_result.succeeded = not run_result.semantic_check.has_err
+      # run_result.succeeded = not run_result.semantic_check.has_err
```

Comment:
intentional to leave this commented out?

Reply:
removed.